mqtt.md

```diff
@@ -152,8 +154,34 @@ This is unnecessary for the X1 series since it already transmits the full object
 }
 ```
 
+- New Request
+
+```json
+{
+    "user_id": "0",
+    "sequence_id": "0",
```

Comment:
The `sequence_id` should be inside pushing, no?

Reply:
Recently, the printer firmware has been updated and the request factor values have changed